test/mailers/hackathons/organizer_summary_test.rb

```diff
@@ -0,0 +1,17 @@
+require "test_helper"
```

Comment:
I would incorporate this into the existing `DigestMailerTest` to match the mailer itself.

Reply:
Could you help me with that? I'm not quite sure what you mean, sorry

Reply:
Organizer summaries are in the existing `DigestMailer`, so use the existing test file as well.

Reply:
Oh I gotcha